src/main/java/com/icthh/xm/tmf/ms/resource/client/AuthorizedFeignClient.java

```diff
@@ -1,14 +1,11 @@
 package com.icthh.xm.tmf.ms.resource.client;
 
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.cloud.openfeign.FeignClientsConfiguration;
 import org.springframework.core.annotation.AliasFor;
 
+import java.lang.annotation.*;
```

Comment:
wildcard imports should be removed, according checkstyle imports orders must be the same as in old version